website/docs/integrator-section/verifiable-document/did/signing-document.md

```diff
@@ -13,16 +13,16 @@ open-attestation sign ./wrapped-documents-did --od ./signed-documents-did --publ
 
 In the example above:
 
-- `public-key` is the public key used [to bind the wallet address with the DNS location](/docs/integrator-section/verifiable-document/did/dns)
-- `key` is the wallet private key, generated when [creating the wallet](/docs/integrator-section/verifiable-document/did/create)
+- `public-key` is the public key used to [bind the wallet address with the DNS location](/docs/integrator-section/verifiable-document/did/dns).
+- `key` is the wallet private key, generated when [creating the wallet](/docs/integrator-section/verifiable-document/did/create).
```

Comment:
Common practice in documentation: 
- If it is a sentence, then it is recommended to add a full stop "." at the end.
- If it is a phrase, it's recommended to end this line without any punctuation. 
- If it ends with a URL, it's recommended to end the line without any "." even it is a complete sentence. 
(because users might copy the URL and paste it in address bar, the "." might be copied and brings an error in the browser)